runtime/binding-tcp/src/main/moditect/module-info.java

```diff
@@ -25,4 +25,7 @@
 
     provides io.aklivity.zilla.runtime.engine.config.ConditionConfigAdapterSpi
         with io.aklivity.zilla.runtime.binding.tcp.internal.config.TcpConditionConfigAdapter;
+
+    provides jakarta.json.spi.JsonProvider
+        with org.leadpony.joy.yaml.YamlProvider;
```

Comment:
This should not be necessary because we don't use modules for the shaded test jar.